src/index.js

```diff
@@ -341,17 +346,25 @@ export function formatCurrency(
         currencyFormatterVerySmall.format(amount),
         locale
       );
-    } else {
+    } else if (price >= 0.0000000001 && price < 0.000001) {
```

Comment:
0.0000000001 is 10 decimal places right? what if we extend it to 0.000000000001 (12) which is the most for `currencyFormatterVeryVerySmall` which is a 12 decimal place

Reply:
@tmlee we can but if we do that,

0.0000000000011.toFixed(12)
'0.000000000001'

Reply:
@tmlee do you think it's better if we get product to advice on this?